.gitignore

```diff
@@ -0,0 +1,19 @@
+# Default ignored files
+/shelf/
+/.idea/workspace.xml
```

Comment:
nit: this one is unnecessary with the one at L19

Reply:
Since this is merged I'll remove it in the next PR, thanks!